static/app/views/dashboards/detail.tsx

```diff
@@ -407,10 +407,15 @@ class DashboardDetail extends Component<Props, State> {
   };
 
   handleChangeFilter = (activeFilters: DashboardFilters) => {
-    const {dashboard, location} = this.props;
+    const {dashboard, location, organization} = this.props;
     const {modifiedDashboard} = this.state;
     const newModifiedDashboard = modifiedDashboard || dashboard;
 
+    trackAnalytics('dashboards2.filter.change', {
+      organization,
+      filter_type: 'release',
+    });
+
```

Comment:
🤔 this is a little confusing, since the method name `handleChangeFilter` seems like it applies to _all_ filter changes, but the analytics call is just for release filters. Could you move this analytics tracking to https://github.com/getsentry/sentry/pull/78688/files#diff-b65b72a83429d6e6ccd3907f351d062b1fee79e4cbeb66751a3cd365ca3e8f33L64

That way all the analytics calls would be together, and it's 100% clear which UI element they're referring to 👀